.github/workflows/build.yml

```diff
@@ -214,6 +214,17 @@ jobs:
 
         popd
 
+        echo " "
+        echo "---- Running measurement_utility_test.sh ... ----"
+        echo " "
+        # Basic verification of measurement_utility.
+        #
+        ./tests/measurement_utility_test.sh
+
+        # Above script will execute a utility which will generate policy_key.py
+        git restore sample_apps/simple_app_python/policy_key.py
```

Comment:
I kicked myself for not thinking about this approach, and so tried to implement this.

However, when implemented, this does not work. `.gitignore` will only ignore files that are not already committed to git. See this:

```
A gitignore file specifies intentionally untracked files that Git should ignore. Files already
       tracked by Git are not affected; see the NOTES below for details.
```

I recall, now, why I needed this file checked-in to git.

In this case, I do need this file in Git, even as just a dummy file. 

Otherwise, if this file is not in git, in a new branch, any attempts to run even just `./example_app.py --help` will fail as follows:

```
cc-sdb-vm:[18] $ PYTHONPATH=../.. ./example_app.py --help
Traceback (most recent call last):
  File "/home/agurajada/Code/certifier-framework-for-confidential-computing/sample_apps/simple_app_python/./example_app.py", line 13, in <module>
    import policy_key
ModuleNotFoundError: No module named 'policy_key'
```

I may be able to overcome this with some Python-gyrations, to avoid failing even when this file `policy_key.py` is missing. But I didn't think that was all that much more a better solution.

Hence, this file remains checked-in and I cannot `.gitignore` it.

Reply:
One simple solution is to put the import statement into `try..except` block and print an error message saying that `policy_key.py` needs to be generated. That way we won't need to put generated files into git.

This can be addressed later though, so go ahead and merge this PR. Thanks.

Reply:
@rgerganov -- I know you said we can try to do this in a follow-on, but I think it's going to be very tedious for the user to "figure out" how to generate `policy_key.py`.

Looking at `--dry-run` output of `run_example.sh`, the message will have to instruct the user to do two things:

- First, run `utilities/cert_utility.exe` to generate `--operation=generate-policy-key-and-test-keys ...  --policy_cert_output_file=policy_cert_file.bin`
- Then, run `utilities/embed_policy_key.exe` to consume as `--input=policy_cert_file.bin`, the generated policy-cert-file
- This will produce the required `--output=../policy_key.py`

I think this is very user un-friendly to have to go thru multiple gyrations so that they can use simple stuff like `--help` interface.

I think we probably just have to live with this dummy file checked-in to git.